src/fabric/src/fabric2_server.erl

```diff
@@ -270,24 +282,31 @@ find_cluster_file([{Type, Location} | Rest]) ->
             ),
             {ok, Location};
         {ok, _} ->
+            ?LOG_ERROR(Msg#{
+                status => permissions_error,
+                details => "CouchDB needs read/write access to FDB cluster file"
+            }),
             couch_log:error(
                 "CouchDB needs read/write access to FDB cluster file: ~s",
                 [Location]
             ),
             {error, cluster_file_permissions};
         {error, Reason} when Type =:= custom ->
+            ?LOG_ERROR(Msg#{status => Reason}),
             couch_log:error(
                 "Encountered ~p error looking for FDB cluster file: ~s",
                 [Reason, Location]
             ),
             {error, Reason};
         {error, enoent} when Type =:= default ->
+            ?LOG_INFO(Msg#{status => enoent}),
             couch_log:info(
                 "No FDB cluster file found at ~s",
                 [Location]
             ),
             find_cluster_file(Rest);
         {error, Reason} when Type =:= default ->
+            ?LOG_WARNING(Msg#{status => Reason}),
```

Comment:
We potentially can call `file:format_error(Reason)` here. It depends on the type of a status field. Would you prefer it to stay atom?

Reply:
I like including the atoms for easier reporting but would be fine to include the `format_error` output in a `details` field.